karma.conf.js

```diff
@@ -12,21 +11,26 @@ module.exports = function setKarmaConfig(config) {
     browsers: ['Chrome'],
     colors: true,
     concurrency: Infinity,
+    coverageReporter: {
+      reporters: [
+        { type: 'lcov', dir: '../coverage/', subdir: '.' },
+        { type: 'json', dir: '../coverage/', subdir: '.' },
+        { type: 'text-summary' }
+      ]
+    },
     files: [{
       pattern: testGlob,
       watched: false,
       served: true,
       included: true
-    }],
-    // files: [testGlob, srcGlob],
+    }, testGlob],
```

Comment:
This looks strange.  The `testGlob` is included in the richer object structure just above; we shouldn't include it twice, should we?